src/features/lightspeed/utils/explorerView.ts

```diff
@@ -29,15 +33,17 @@ export function getWebviewContentWithLoginForm(
       <title>Ansible Lightspeed Explorer!</title>
     </head>
     <body>
-    Welcome to Ansible Lightspeed for Visual Studio Code.\nExperience smarter automation using Ansible Lightspeed with watsonx Code Assistant solutions for your playbook. <a href="https://www.redhat.com/en/engage/project-wisdom">Learn more</a><br />Let's simplify your workflow by connecting VS Code with Ansible Lightspeed.<br />
-    <form id="playbook-explanation-form">
-      <div class="component-container">
-        <section class="component-section">
-          <vscode-button id="lightspeed-explorer-connect">Connect</vscode-button>
+    <div id="lightspeedExplorerView">
+      Welcome to Ansible Lightspeed for Visual Studio Code.\nExperience smarter automation using Ansible Lightspeed with watsonx Code Assistant solutions for your playbook. <a href="https://www.redhat.com/en/engage/project-wisdom">Learn more</a><br />Let's simplify your workflow by connecting VS Code with Ansible Lightspeed.<br />
+      <form id="playbook-explanation-form">
+        <div class="button-container">
+          <section class="component-section">
+            <vscode-button id="lightspeed-explorer-connect">Connect</vscode-button>
+          </section>
         </section>
```

Comment:
I think this `</section>` needs to be changed to `</div>`, which corresponds to line 39.